Dockerfile

```diff
@@ -6,6 +6,8 @@ MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
 USER root
 
+RUN apt-get update
```

Comment:
Do you mean add it before each `apt-get install` command?  E.g.

    RUN apt-get update && apt-get install -y julia libnettle4 && apt-get clean

What is your rationale for doing this?  Is it to minimise the size of the docker image?